src/components/pill/index.js

```diff
@@ -0,0 +1,146 @@
+import { html, css, LitElement, unsafeCSS } from 'lit';
+import { createRef, ref } from 'lit/directives/ref.js';
+import styles from './styles.scss';
+import { classMap } from 'lit/directives/class-map.js';
+import { TYPE } from './enums';
+
+export class VlPill extends LitElement {
+  static get EVENTS() {
+    return {
+      close: 'close',
+      check: 'check',
+    };
+  }
+
+  static get styles() {
+    return [
+      css`
+        ${unsafeCSS(styles)}
+      `,
+    ];
+  }
+
+  static get properties() {
+    return {
+      classes: {},
```

Comment:
Classes hoeft geen property te zijn, je kan gewoon een const aanmaken in de render functie.

Reply:
Fixed.